src/molecules/Header.tsx

```diff
@@ -7,6 +7,7 @@ import { NavMenu } from './NavMenu'
 import { Logo } from '../atoms/Logo'
 import { Bar } from './Bar'
 import routes from '../routes'
```

Comment:
I don't think you use this `routes` import in this file.

Reply:
I used it here :
```
<Link to={routes.home}>
```